SYCL/ESIMD/regression/math_const_fix_test.cpp

```diff
@@ -23,6 +23,7 @@
 
 constexpr auto sycl_write = sycl::access::mode::write;
 #define SIMD 16
+int errCnt = 0;

```

Comment:
Thank you for the additional fix.
Sorry, but one more minor comment: Can you please name it ErrCnt (or ErrorCount) to follow clang coding std that asks the variable to start with a capital letter and functions with lower case: int MyVariable; int myFunction();.

Reply:
Or alternatively you can follow the other (C++ std) that uses all lower cases with underscores used in other places of this file: i.e. err_cnt;